### PR TITLE
Add Challenge exporting to JSON functionality

### PR DESCRIPTION
**Pull request by piotrekpasciak**
Monday ticket:
https://aicrowd.monday.com/boards/363093650/pulses/447286098

Github Issue:
https://github.com/AIcrowd/AIcrowd/issues/788

Blocked until questions in Github Issue gest answered.

### CODE REVIEW

File: spec/requests/challenges_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+      it 'sends json file in response' do
+        get export_challenge_path(challenge)
+
+        expect(response).to have_http_status :ok
+        expect(response.header['Content-Type']).to eq 'application/json'
+
+        challenge_data = JSON.parse(response.body)
+
+        expect(challenge_data['challenge']).to eq 'Challenge Title'
```
Comment (nemaniarjun) [inline spec/requests/challenges_controller_spec.rb]: 💯

File: app/models/challenge.rb [hunk 2]
```diff
@@ -40,8 +123,6 @@ class Challenge < ApplicationRecord
            class_name: 'ParticipantChallengeCount'
   has_many :challenge_registrations,
            class_name: 'ChallengeRegistration'
-  has_many :challenge_organizer_participants,
```
Comment (alkeshrails) [inline app/models/challenge.rb]: Why we require to remove this association?

Comment (piotrekpasciak) [reply app/models/challenge.rb]: This relation is broken, and was causing issues for me. I'll double check if I can make it work easly.

Comment (piotrekpasciak) [reply app/models/challenge.rb]: @alkeshrails 
ChallengeOrganizerParticipant doesn't have `challenge_id` foreign key. This association simply cannot work.
https://github.com/AIcrowd/AIcrowd/blob/master/db/schema.rb#L990

**Pull request merged**